pipeline-braille-scripts/dtbook-to-pef/src/main/resources/xml/xproc/dtbook-to-pef.xpl

```diff
@@ -571,50 +124,16 @@ content at the beginning of every other volume, include the following additional
     <!-- ============= -->
     <!-- DTBOOK TO PEF -->
     <!-- ============= -->
-    <px:dtbook-to-pef.convert default-stylesheet="http://www.daisy.org/pipeline/modules/braille/dtbook-to-pef/css/default.css">
+    <px:dtbook-to-pef.convert>
         <p:input port="source">
             <p:pipe step="main" port="source"/>
         </p:input>
-        <p:with-option name="stylesheet" select="$stylesheet"/>
-        <p:with-option name="transform" select="$transform"/>
```

Comment:
I like it that you changed the options to parameters but maybe `stylesheet` and `transform` should stay options because they are used as options for `css:inline` and `px:transform` respectively and there is no reason to pass them along as parameters (sass variables or translation parameters).